augur/traits.py

```diff
@@ -174,10 +174,16 @@ def run(args):
             continue
 
         for node in T.find_clades():
-            mugration_states[node.name][column] = node.__getattribute__(column)
+            mugration_states[node.name][column] = getattr(node, column)
+
             if args.confidence:
-                mugration_states[node.name][column+'_confidence'] = node.__getattribute__(column+'_confidence')
-                mugration_states[node.name][column+'_entropy'] = node.__getattribute__(column+'_entropy')
+                confidence = getattr(node, f"{column}_confidence", None)
+                if confidence:
+                    mugration_states[node.name][f"{column}_confidence"] = confidence
+
+                entropy = getattr(node, f"{column}_entropy", None)
+                if entropy:
+                    mugration_states[node.name][f"{column}_entropy"] = entropy
```

Comment:
Is it possible to have falsy values for `confidence` and `entropy` and do we want to keep them? If so, we should check specifically that they are not `None`.

Reply:
Good call, @joverlee521! 53cf195 fixes this.